seatunnel-connectors-v2/connector-file/connector-file-base/src/test/java/org/apache/seatunnel/connectors/seatunnel/file/Reader/ExcelReadStrategyTest.java

```diff
@@ -15,8 +15,9 @@
  * limitations under the License.
  */
 
-package org.apache.seatunnel.connectors.seatunnel.file.writer;
+package org.apache.seatunnel.connectors.seatunnel.file.Reader;
```

Comment:
```suggestion
package org.apache.seatunnel.connectors.seatunnel.file.reader;
```

Reply:
![image](https://github.com/user-attachments/assets/7e10f6fe-7202-45dd-bc40-15bfe9820adf)
Excel stores the double type of the data of the time type at the bottom layer, so using double to convert back to the Date and DateTime types can now pass all test cases.
And added the string matching recognition time formatting option of yyyy/M/d and yyyy-M-d in DateTimeUtils and DateUtils